src/sdk_codes.ts

```diff
@@ -0,0 +1,171 @@
+import { Logger } from './log';
+import { Target } from './types';
+
+interface SDKCodeMessages {
+  [key: number]: string;
+}
+
+const sdkCodes: SDKCodeMessages = {
+  // SDK_INIT_1xxx
+  1000: 'The SDK has successfully initialized',
+  1001: 'The SDK has failed to initialize due to an authentication error - defaults will be served',
+  1002: 'The SDK has failed to initialize due to a missing or empty API key - defaults will be served',
+  // SDK_AUTH_2xxx
+  2000: 'Authentication was successful',
+  2001: 'Authentication failed with a non-recoverable error',
+  2002: 'Authentication attempt',
+  2003: 'Authentication failed and max retries have been exceeded',
+  // SDK_CLOSE_3xxx
+  3000: 'Closing SDK',
+  3001: 'SDK Closed successfully',
+  // SDK_POLL_4xxx
+  4000: 'Polling started, intervalMs:',
+  4001: 'Polling stopped, reason:',
+  // SDK_STREAM_5xxx
+  5000: 'SSE stream successfully connected',
+  5001: 'SSE stream disconnected, reason:',
+  5002: 'SSE event received: ',
```

Comment:
Looks like this should have details of the event, but it'll just have 2 spaces instead.

Reply:
Whoops, forgot to update the string to remove the colon as I opted not to log the event - it's too noisy. This code will only be logged in a `data` event on the SSE stream, the actual event is then logged in due course.